mod1/LazyTeenager.java

```diff
@@ -16,17 +16,17 @@ public static void main(String[] args) {
         // while loop
         while (roomClean == false) {
             timesAsked ++;
+            precentChance = 100 / (timesAsked * 5);
```

Comment:
There's a couple wrong things here and you should debug through this to see what it does

Youre getting a percent chance - by getting a fraction- which is fine, but you are using an `int` here so you would be cutting off decimal points in case you needed precision here. For example, if `timesAsked` is 3, you would get `100 / (3 * 5) = 7.5 which would go to 7. 

Then you're generating a random number with the upperbound that you got from the previous function. So you're saying, give me a random number between 0 and percentChance. The chance of the work being done does go up, since the more times are asked, the smaller percent chance gets. This is _sort of_ correct but unintuitive.

What should be done is getting a random number from 0-100 with `nextint(100) + 1` and then comparing that against `timesAsked * 5` to see which is greater.

Reply:
I did that on perpose so it whouldent genarate random decimals

Reply:
ok let's code through this together and this is a good opportunity to debug through it